FlatCache.podspec

```diff
@@ -8,5 +8,5 @@ Pod::Spec.new do |spec|
   spec.source       = { :git => 'https://github.com/GitHawkApp/FlatCache.git', :tag => '#{s.version}' }
```

Comment:
Here we specific the source files are from Git Tag. But we don't have any tag in this Repo. Also the pod is not released yet. Checked GitHawk and it is using `branch` option and directly points to master branch which works fine. But still we should be tagging the versions though it is pre release

Reply:
This was fixed 😄